pheps/phep-0003.md

```diff
@@ -0,0 +1,374 @@
+```
+PHEP: 3
+Title: PyHC Python & Upstream Package Support Policy
+Author: Shawn Polson <pyhc-confidential@lasp.colorado.edu> <https://orcid.org/0000-0003-0619-5745>
+Discussions-To: https://github.com/heliophysicsPy/standards/pull/29
+Revision: 1
+Status: Draft
+Type: Standards Track
+Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+Created: 06-Jun-2024
+Post-History: 06-Jun-2024, 11-Jun-2024, 02-Jul-2024, 17-Jul-2024, 23-Jul-2024
+```
+
+# Abstract
+<a name="abstract"></a>
+This PHEP recommends that all projects across the PyHC ecosystem adopt a common time-based policy for support of dependencies, inspired by [SPEC 0](https://scientific-python.org/specs/spec-0000/). 
```

Comment:
Is "recommend" the right word? This gets into the question of applicability of PHEPs which I think we're still feeling out as a community, but I'd suggest stronger wording:
"This PHEP establishes a common time-based policy for support..."

Reply:
This is something Julie and I should discuss but I suspect we'll end up leaving the "recommend" wording. This PHEP was originally written as more of a "must" but the first major pushback I got in the earlier comments were people requesting I soften the policy; hence why `We decided this policy has to be a "should" not a "must."` is the second bullet under the `Resolved questions and comments` section of this PR's description. It also more closely follows SPEC 0's wording which uses "recommend."

Reply:
I'll add that even if I started this first sentence with "This PHEP establishes a common time-based policy for support...", the word "recommends" occurs immediately in the next line. So I'd have to change that wording and all subsequently-related statements to match, which again goes against what we'd previously decided.

Reply:
Not a hill I intend to die on. But maybe worth putting explicitly in the rejected ideas? E.g. "it was considered making this a requirement rather than a recommendation but the community argued against because xyz"?

Reply:
Done.